### PR TITLE
chore: Update translations

### PR DESCRIPTION
**Pull request by RuffleBuild**
New Crowdin pull request with translations 🎉

### CODE REVIEW

File: web/packages/extension/assets/_locales/zh_TW/messages.json
Comment (torokati44) [inline web/packages/extension/assets/_locales/zh_TW/messages.json:1]: Ah, these must be addressing https://github.com/ruffle-rs/ruffle/pull/18354#discussion_r1816791643.

File: desktop/packages/linux/rs.ruffle.Ruffle.desktop [hunk 1]
```diff
@@ -23,6 +23,7 @@ GenericName[ja]=Flashプレイヤー
 GenericName[ko]=플래시 플레이어
 GenericName[nl]=Flash speler
 GenericName[pl]=Odtwarzacz Flash
+GenericName[pt_PT]=Flash Player
```
Comment (kjarosh) [inline desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: Not sure about this translation: "Player" is translatable here, it should be consistent with the keywords (where "reprodutor" is used).

Comment (torokati44) [reply desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: So it's capitalized in the source because ... title-case? 🤔

Comment (kjarosh) [reply desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: yes

Comment (torokati44) [reply desktop/packages/linux/rs.ruffle.Ruffle.desktop:26]: I see - made a comment about it on Crowdin, and undone this translation.

**Review by torokati44 — APPROVED** [review]
Would merge if there was a way to nudge the checks to actually run again from mobile.

**Pull request merged**